README.md

```diff
@@ -2,11 +2,23 @@
 
 <img alt="ORBIT Logo" src="logos/orbit_logo_simple.png" align="right" width="520" >
 
-The **O**pen **R**untime **B**inary **I**nstrumentation **T**ool, is a standalone **C/C++** profiler for Windows and Linux. Its main purpose is to help developers visualize the execution flow of a complex application.
-
-The key differentiator with many existing tools is that no alteration to the target process is necessary. Orbit does not require you to change a single line of code. It doesn't require you to recompile or even relaunch the application you want to profile. Everything is done seamlessly, right when you need it. It requires zero integration time and zero iteration time.
-
-Orbit combines sampling and dynamic instrumentation to optimize the profiling workflow. Sampling can quickly identify interesting functions to instrument. Dynamic instrumentation results in exact function entry and exit information which is presented in the form of per-thread hierarchical call graphs.  Scheduling events are also shown to visualize when a thread was running and on what core.
+The **O**pen **R**untime **B**inary **I**nstrumentation **T**ool, is a
+standalone **native** application (C, C++, Rust, go, ...) profiler for Windows
+and Linux. Its main purpose is to help developers visualize the execution flow
+of a complex application.
```

Comment:
"Its main purpose is to help developers visualize the execution flow
of a complex application."
I know it was already there but I'm not sure I would write this as the main purpose of a profiler.

Reply:
I guess this is not meant to be the main purpose of _a_ profiler, but the main purpose of Orbit. Given that we do not support low-level CPU profiling (hardware counters), it's not a bad description of what you can do with Orbit (but debatable, definitely open for suggestions).

Reply:
What I'm saying is that this sentence says that the *main* purpose of (not simply what you can do with) Orbit, a profiler, is something different than profiling ("visualizing the execution flow"). Maybe we intend something different with "execution flow", but to me that's something for an IDE or a debugger. I think that something like "visualize what functions a complex application spends most of its time executing" or "visualize how a complex application utilizes CPU resources" or "identify the performance bottlenecks" or anything along these lines would be more accurate.

Reply:
I kinda like "identify the performance bottlenecks of a complex application", as this is also vague enough to include the *complex execution flow* part.

Reply:
I don't find it questionable that "visualizing the execution flow" is one of the main things one can do with Orbit, but I would probably add "with exact timing information" there. Features such as manual instrumentation, dynamic instrumentation, thread states and dependencies, and all the GPU features, visualized in the capture window are exactly that. Or, maybe, a different perspective here is that Orbit is not just a profiler?

And I'm not sure if being more vague here is better. Maybe we should just explicitly list the main things one can do
"visualize the execution flow, with precise timing information, of a complex application", "find CPU bottlenecks using callstack sampling", "find GPU bottlenecks using various GPU execution timing data".

Reply:
The premise is that this sentence starts with "its main purpose". What you mention as part of "visualizing the execution flow" is either a subset of the means to achieve that purpose, or doesn't sound very profiler-like, depending on whether more or less detail is added to those words. Since the sentence probably wanted to be brief, I think this could all fit well in the paragraphs starting at line 10 and 16.

Reply:
Added a second sentence.